includes/upgrades/upgrade-4.1.7.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Migrate subscription sort order data (runs once)
+ *
+ * This function ensures that all existing subscription packs have a default sort order
+ * set if they don't already have one.
+ *
+ * @since WPUF_SINCE
+ *
+ * @return void
+ */
+function wpuf_upgrade_4_1_7_subscription_sort_order_migration() {
+
+    $migration_version = get_option( 'wpuf_subscription_sort_order_migration', '0' );
+    $current_version = '1.0';
```

Comment:
no need to manager version separately. in `includes/Admin/Upgrades.php` we are checking if migration is completed or not